test/pummel/test-process-hrtime.js

```diff
@@ -9,7 +9,7 @@ assert(Array.isArray(start));
 
 // busy-loop for 2 seconds
 var now = Date.now();
-while (Date.now() - now < 2000);
+while (Date.now() - now < 2000) {}
```

Comment:
This style shouldn't actually be enforced. You can do useful things inside `while();` and `for(;;);` statements.